cypress/integration/pages/testsForAllPages.js

```diff
@@ -16,281 +15,31 @@ export const testsThatFollowSmokeTestConfigforAllPages = ({
-                      break;
-                    case 'storyPage':
-                      description = body.promo.summary;
-                      title = body.promo.headlines.headline;
-                      break;
-                    case 'idxPage':
-                      description = body.metadata.summary;
-                      title = body.metadata.title;
-                      break;
-                    default:
-                      description = '';
-                      title = '';
-                  }
-                  /* Note that if updating these, also do the same for errorPage404/tests.js */
-                  const pageTitle = `${title} - ${
-                    appConfig[config[service].name][variant].brandName
-                  }`;
-
-                  cy.get('head').within(() => {
-                    cy.title().should('eq', pageTitle.replace(/ +/g, ' '));
-                    cy.get('meta[property="og:description"]').should(
-                      'have.attr',
-                      'content',
-                      description,
-                    );
-                    cy.get('meta[property="og:title"]').should(
-                      'have.attr',
-                      'content',
-                      pageTitle,
-                    );
-                    cy.get('meta[name="twitter:description"]').should(
-                      'have.attr',
-                      'content',
-                      description,
-                    );
-                    cy.get('meta[name="twitter:title"]').should(
-                      'have.attr',
-                      'content',
-                      pageTitle,
-                    );
-                  });
-                },
-              );
-            }
-          });
-        }
-      }
-      /* End of block (pageType !== 'errorPage404') */
-
-      it('should have dir matching service config', () => {
-        cy.get('html').and(
-          'have.attr',
-          'dir',
-          appConfig[config[service].name][variant].dir,
+    describeForEuOnly('Consent Banners', () => {
+      it('have correct translations', () => {
+        cy.contains(
+          appConfig[config[service].name][variant].translations.consentBanner
+            .privacy.title,
+        );
+        cy.contains(
+          appConfig[config[service].name][variant].translations.consentBanner
+            .privacy.reject,
+        );
+        cy.contains(
+          appConfig[config[service].name][variant].translations.consentBanner
+            .privacy.accept,
+        ).click();
+        cy.contains(
+          appConfig[config[service].name][variant].translations.consentBanner
+            .cookie.title,
+        );
+        cy.contains(
+          appConfig[config[service].name][variant].translations.consentBanner
+            .cookie.reject,
+        );
+        cy.contains(
+          appConfig[config[service].name][variant].translations.consentBanner
+            .cookie.accept,
```

Comment:
This was removed as part of https://github.com/bbc/simorgh/pull/6925 (which was merged earlier today hence merge conflict 😿 ), so also needs to be removed here.